packages/react/src/Autocomplete/AutocompleteMenu.tsx

```diff
@@ -381,9 +382,9 @@ function AutocompleteMenu<T extends AutocompleteItemProps>(props: AutocompleteMe
             </ActionList>
           ) : emptyStateText !== false && emptyStateText !== null ? (
             enabled ? (
-              <Box className={classes.EmptyStateWrapper}>{emptyStateText}</Box>
+              <Announce className={classes.EmptyStateWrapper}>{emptyStateText}</Announce>
```

Comment:
Need to use `LiveRegion` or `Debounce` because this does not provide feedback if a user continues to type.